frontend/go.mod

```diff
@@ -0,0 +1,3 @@
+module front-end-cloud-calc
+
+go 1.19
```

Comment:
Downgrade to 1.18 to be the same as backend  and for Devs who don't have 1.19 yet

Reply:
Too slow